src/main/java/com/carrotcreative/recyclercore/widget/ProgressRecyclerViewLayout.java

```diff
@@ -60,4 +60,13 @@ public void setAdapter(RecyclerView.Adapter adapter)
         mProgressBar.setVisibility(GONE);
     }
 
+    /**
+     *
+     * @return The instance of recycler view in the layout.
+     */
+    public RecyclerView getRecyclerView()
```

Comment:
Note to self:
Maybe I should remove this method - As we probably do not want to give the developer direct access to the recyclerview. As when they have direct access - they can bypass the calls to `setLayoutManager` and `setAdapter`.